settings/defaults.json

```diff
@@ -0,0 +1,34 @@
+{
+  "vessels": {
+    "self": {
+      "name": "Aava",
```

Comment:
Details really, but the default vessel name could be "default" or something, and the uuid something very obvious default, instead of using someone's actual values.